### PR TITLE
Bug 1655950 - Sets a dummy value to  get into reload-required status as required

### PR DESCRIPTION
**Pull request by josejulio**
This allows to safely set undefined to io-threads and task-max-threads, other way we would
  get a IllegalArgumentException

### CODE REVIEW

**Comment by josejulio** [comment]
@loleary  @burmanm can you comment on this approach?

File: modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/WorkerComponent.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.rhq.modules.plugins.wildfly10;
```
Comment (loleary) [inline modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/WorkerComponent.java]: Missing copyright/license header.

**Comment by loleary** [comment]
I think this approach is dangerous but it does seem to be a valid workaround.

My reasoning is that we are applying the workaround blindly -- Regardless of version or what configuration attributes are actually changing. That said, it does seem that all the configuration attributes for an io worker resource require reload. Meaning that if any configuration changes are made, reload would be required. Therefore, this does make sense to me. I am curious to what @burmanm thinks.

File: modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/WorkerComponent.java (hunk 1 above)
Comment (josejulio) [reply modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/WorkerComponent.java]: Thanks

**Comment by burmanm** [comment]
@loleary We talked about this in the IRC, a short recap here. For other values (like the task-keepalive mentioned in the ticket) we should do the update without this trick and instead just remove these values which we can't update from the batch update sent to the EAP. For these values which we can't currently unset in the EAP, we would first contact the EAP team and see if it's a bug that should be fixed in the actual product. And we'll wait a bit for their reply before merging any workarounds.

**Comment by josejulio** [comment]
Closing in favor of #377

**Pull request closed without merging**